scripts/deploySpace1.ts

```diff
@@ -0,0 +1,106 @@
+import fs from 'fs';
```

Comment:
Why `deploySpace1`? Could we find a more explicit name maybe?

Reply:
thats a old file, deleted it